ATMoviess/ATMoviess/Models/UpcomingMoviesModel.cs

```diff
@@ -0,0 +1,99 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace ATMoviess.Models
+{
+    public class UpcomingMoviesModel
+    {
+        public List<Result> Results { get; set; }
+        public int Page { get; set; }
+        public int Total_results { get; set; }
+        public Dates Dates { get; set; }
+        public int Total_pages { get; set; }
+    }
+
+    public class Result
+    {
+        public int Vote_count { get; set; }
+        public int Id { get; set; }
+        public bool Video { get; set; }
+        public double Vote_average { get; set; }
+        public string Title { get; set; }
+        public double Popularity { get; set; }
+        private string _poster_path;
+        public string Poster_path
+        {
+            get
+            {
+                if (string.IsNullOrEmpty(_poster_path))
+                    return "noposter.png";
+
+                return "http://image.tmdb.org/t/p/w500" + _poster_path;
```

Comment:
Better to keep magic strings as `const` values declared elsewhere.
It is easier to maintain and easier to make configurable.

Reply:
I see, thanks for the feedback!